### ISSUE TITLE
help doesn't work if you don't have a package.json

### ISSUE THREAD

**Issue by sam-github** [watilde/dep#5]
I think it should!

```
~ % dep --help
module.js:491
    throw err;
    ^

Error: Cannot find module '/Users/sam/package.json'
...
~ % dep help
module.js:491
    throw err;
    ^

Error: Cannot find module '/Users/sam/package.json'
...
```

**Comment by watilde** [comment]
This is obviously a bug! Thanks for your report and I will fix quickly.

**Comment by watilde** [comment]
#6 was released as `v0.14.1` and the issue was fixed. Thanks!
